constants.py

```diff
@@ -25,7 +25,7 @@
 CONFIG_BUFFER_SIZE = 'buffer_size'
 CONFIG_DEFAULT_VALUES = {
     CONFIG_MUTE: False,
-    CONFIG_QUALITY: "best",
+    CONFIG_QUALITY: ["720p", "480p", "360p", "160p"],
```

Comment:
Hmm, good thought but these are just the default values for the config and shouldn't be used anywhere else. Also there's probably more than 40+ stream options we would have to save in the enums to have enums for all quality options, and even then we're not sure if we have all because the streamlink plugins might decide to change... For example for many plugins the 720 option name is `720`, for others it's `720p` and some other `http_720p` or `why_did_we_let_the_plugin_writers_decide_what_name_to_give_the_quality_options_720`

So for consistency I recommend we leave all quality options as strings in the application.